### DISCUSSION TITLE
Add support for long term statistics to aggregated sensors

### DISCUSSION THREAD

**Discussion by MadD0c** [jseidl/magic-areas#257] [Ideas / Feature Requests]
It would be a handy feature to add in support for Long Term Statistics with HA for Aggregated Sensors. This would make it easier to manage and monitor long term.

According to this https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics a state_class would need to be set, most likely to measurement.

**Comment by jseidl** [comment]
Cool, thanks for the idea!

**Comment by jseidl** [comment]
This is coming up next release!
